x-pack/plugins/security_solution/public/types.ts

```diff
@@ -148,5 +148,5 @@ export interface StartedSubPlugins {
   management: ReturnType<Management['start']>;
   landingPages: ReturnType<LandingPages['start']>;
   cloudSecurityPosture: ReturnType<CloudSecurityPosture['start']>;
-  threatIntelligence: ReturnType<ThreatIntelligence['start']>;
```

Comment:
lets add a word or two why this is optional :)

Reply:
good idea, I can add a comment that this is a temporary change!

Reply:
added!